build/webpack.base.conf.js

```diff
@@ -29,19 +29,38 @@ module.exports = function baseConfig(env) {
     },
     module: {
       rules: [
-        // Source map loader to map typescript source-maps
+        // Source map loader to map typescript and babel source-maps
+        // for debugging and coverage purposes
         {
           enforce: 'pre',
           test: /\.js%/,
           loader: 'source-map-loader'
         },
-        // ts-loader to compile typescript using tsc
+        // tslint loader to lint files during build
+        {
+          test: /\.ts$/,
+          enforce: 'pre',
+          loader: 'tslint-loader',
```

Comment:
Another thing we could have done was have the `npm run test` command also run the lint checker alongside with the tests

Reply:
This does integrate it with the `npm test` command. It will break the test and production builds, whereas the development build will still succeed with broken style for those who like to fix their linting at the end of their PR

Reply:
Ah I see, great!